src/Uno.UI/UI/Xaml/Media/Animation/RepositionThemeAnimation.cs

```diff
@@ -0,0 +1,14 @@
+namespace Windows.UI.Xaml.Media.Animation;
+
+#if __ANDROID__ || __IOS__ || IS_UNIT_TESTS || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
+[global::Uno.NotImplemented]
+#endif
+public partial class RepositionThemeAnimation : Timeline, ITimeline
+{
+	// RepositionThemeAnimation is currently not implemented.
+	// But we need to call OnCompleted() so that whenever RepositionThemeAnimation is
+	// included in a Storyboard, we want to allow the Storyboard to fire Completed event
+	// This is important especially when RepositionThemeAnimation is part of a transition Storyboard.
+	// Visual state setters aren't executed unless the transition is completed.
+	void ITimeline.Begin() => OnCompleted();
```

Comment:
That's an interesting workaround. There are other animations like this one that likely would benefit from the same change (in another PR).

Reply:
Yeah I thought of other animations, but found `RepositionThemeAnimation` to be the most commonly used one and wanted to quickly get #13303 fixed. Other non-implemented animations are very rarely used. We can follow-up if needed in another PR.